packages/superset-ui-core/src/query/buildQueryObject.ts

```diff
@@ -20,7 +20,7 @@ export default function buildQueryObject<T extends QueryFormData>(
 ): QueryObject {
   const {
     annotation_layers = [],
-    extra_form_data = {},
```

Comment:
why not leave it?

Reply:
I was getting a weird TS error - it was moved down to line 42 in the form of `extra_form_data || {}`.

Reply:
👍